libvirt/stream.go

```diff
@@ -18,6 +20,7 @@ func (sio *StreamIO) Write(p []byte) (int, error) {
 	return sio.Stream.Send(p)
 }
 
+// Close  StreamIO
```

Comment:
One space too many. Also, the comment could provide a bit more information, e.g. "Close closes the stream."